files/KoreBuild/modules/sharedsources/module.targets

```diff
@@ -32,7 +32,7 @@ that matches "$(RepositoryRoot)/shared/*.Sources".
 
   <Target Name="_SetSharedSourcesProperties">
     <PropertyGroup>
-      <_SharedSourcesPackageProperties>PackageOutputPath=$(BuildDir);RepositoryRoot=$(RepositoryRoot);BuildNumber=$(BuildNumber);</_SharedSourcesPackageProperties>
+      <_SharedSourcesPackageProperties>PackageOutputPath=$(BuildDir);RepositoryRoot=$(RepositoryRoot);ImportDirectoryBuildProps=false;BuildNumber=$(BuildNumber);</_SharedSourcesPackageProperties>
```

Comment:
Would adding an empty `Directory.Build.props` work instead?

Reply:
Tried that: https://github.com/aspnet/BuildTools/commit/cafc5de7d2eb41fe85302f034e1eab2c7ce8d335